pkg/installer/args.go

```diff
@@ -26,10 +26,10 @@ import (
 )
 
 const (
-	csiProvisionerImage      = "csi-provisioner:v3.4.0"
-	nodeDriverRegistrarImage = "csi-node-driver-registrar:v2.6.3"
-	livenessProbeImage       = "livenessprobe:v2.9.0"
-	csiResizerImage          = "csi-resizer:v1.7.0"
+	csiProvisionerImage      = "csi-provisioner@sha256:704fe68a6344774d4d0fde980af64fac2f2ddd27fb2e7f7c5b3d8fbddeae4ec8"
```

Comment:
can we add a comment on the top to know what side car version these hashes corresponds to?

Reply:
done